build.sbt

```diff
@@ -64,6 +64,8 @@ lazy val core = (project in file("."))
       Libraries.fs2Core,
       Libraries.fs2Io,
       Libraries.guava,
+      Libraries.skunk,
```

Comment:
what about doobie? 🤔

Reply:
TL;DR: "use Doobie for general-purpose database interaction and multi-database support and use Skunk if you're committed to PostgreSQL and want the most type-safe, performant, and PostgreSQL-optimized solution."

I think is worth a shot for our use case. 
(anyway, since the syntax/api usage is pretty similar, switching later is not that big impact)

https://www.notion.so/SQL-libraries-160a94bd4b4f8011b72bfb43caa0fea8